Packages/vcs/Lib/configurator.py

```diff
@@ -19,128 +21,61 @@
 
 import copy
 
-def sync_template(src, target):
```

Comment:
Whitespace fixes; sorry, I know I should have done that in another branch, but my linter was re-enabled recently and I couldn't un-see the errors.